software/ADPL_electron/ADPL_electron.ino

```diff
@@ -60,8 +60,8 @@ void loop() {
     temp_count = read_temp(temp_count);
 
     if ((currentTime - last_publish_time) > PUBLISH_DELAY) {
-        sprintf(temps_str,"HXCI:%.1f,HXCO:%.1f,HTR:%.1f,HXHI:%.1f,HXHO:%.1f",
-                tempHXCI.temp, tempHXCO.temp, tempHTR.temp, tempHXHI.temp, tempHXHO.temp);
+        sprintf(temps_str,"HXCI:%.1f,HXCO:%.1f,HTR:%.1f,HXHI:%.1f,HXHO:%.1f,VALVE:%d",
```

Comment:
Fantastic, I'll make the change and merge to master after the corresponding server changes are made :).